packages/mrc-ide-covidsim/src/imperial-params.ts

```diff
@@ -274,6 +273,31 @@ export function assignAdminParameters(p: {}, subregionName: string) {
   p['Codes and country/province names for admin units'] = adminUnitNameLookup
 }
 
+// exported for testing
+export function setParameterFamilyTo0(
+  p: {},
+  familyName: string,
+  periodCount: number
+) {
+  familyName = familyName.toLowerCase()
+  Object.entries(p)
+    .filter(([key]) => key.toLowerCase().includes(familyName))
+    .forEach(([key, value]) => {
+      if (Array.isArray(value)) {
+        // the top-level array should change size only if this is an _over time_ parameter
+        const newLen = /over time/i.test(key) ? periodCount : value.length
+        p[key] = new Array(newLen).fill(0)
```

Comment:
Stylistic suggestion: pull out a function `zeroes(len: number): number[]` and use it here and below.

Reply:
sure